plugins/binanceExchange_ws_test.go

```diff
@@ -89,3 +103,70 @@ func Test_binanceExchangeWs_GetOrderBook(t *testing.T) {
 		assert.True(t, ob.Bids()[0].Volume.AsFloat() > 0)
 	}
 }
+
+func Test_binanceExchangeWs_GetLatestTradeCursor(t *testing.T) {
+
+	if testing.Short() {
+		return
+	}
+
+	startIntervalSecs := time.Now().Unix()
+
+	testBinanceExchangeWs, err := makeBinanceWs(emptyAPIKeyBinance)
```

Comment:
does this test pass without using valid API keys? -- I think this test should fail without valid keys but I'm not sure (see comment above about throwing an error when we can't open up the cursor stream)